ansible/roles/installation/steamvr/tasks/main.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Update package lists
+  apt: update_cache=yes
+  become: yes
+  
+- name: Intall needed libs for steamvr
+  apt: name={{ item }} install_recommends=no
+  become: yes
+  with_items:
+    - curl
+    - ca-certificates
+    - lib32gcc1
+    - binutils
+    - xdg-utils
+    - software-properties-common
+    - sudo
+    - libvulkan1
+    - usbutils
+    - libcap2-bin
+
+- name: Update and upgrade apt packages
+  become: yes
+  apt:
+    upgrade: dist
+    update_cache: yes
+
+- name: Gather package facts
+  package_facts:
+    manager: apt
+
+- name: Install libudev0_175 from the internet if absent
+  apt:
+    deb: http://mirrors.kernel.org/ubuntu/pool/main/u/udev/libudev0_175-0ubuntu9_amd64.deb
+    state: present
+  become: yes
+  when: '"libudev0" not in ansible_facts.packages'
+
+- name: Create steamcmd directory
+  file:
+    path: "/home/{{ user }}/steamcmd"
+    state: directory
+
+- name: Unarchive steamcmd_linux from the internet
+  unarchive:
+    src: https://steamcdn-a.akamaihd.net/client/installer/steamcmd_linux.tar.gz
+    dest: "/home/{{ user }}/steamcmd"
+    remote_src: yes
+    creates: /home/{{ user }}/steamcmd/steamcmd.sh
+
+- name: Install and update steamcmd
+  shell: "/home/{{ user }}/steamcmd/steamcmd.sh +login anonymous +quit"
+  args:
+    creates: "/home/{{ user }}/steamcmd/package/steam_cmd_linux.installed"
+
+- name: Create sdk32 directory
+  file:
+    path: "/home/{{ user }}/.steam/sdk32"
+    state: directory
+
+- name: Create sdk64 directory
+  file:
+    path: "/home/{{ user }}/.steam/sdk64"
+    state: directory
+
+- name: Copy 32bit steamclient
+  copy:
+    src: "/home/{{ user }}/steamcmd/linux32/steamclient.so"
+    dest: "/home/{{ user }}/.steam/sdk32"
+    remote_src: yes
+
+- name: Copy 64bit steamclient
+  copy:
+    src: "/home/{{ user }}/steamcmd/linux64/steamclient.so"
+    dest: "/home/{{ user }}/.steam/sdk64"
+    remote_src: yes
+
+- name: Install and update steamvr
+  command: >
+    /home/{{ user }}/steamcmd/steamcmd.sh +login shadow_software_1 shadow_software
+    +force_install_dir /home/{{ user }}/.steam +app_update 250820 -beta beta validate +quit
+  args:
+    creates: /home/{{ user }}/.steam/bin/vrstartup.sh
+
+- name: Fix setcap error in vrstartup.sh
+  become: yes
```

Comment:
try to keep `become` as closer to bottom as possible.

Reply:
done